draft-ietf-acme-acme.md

```diff
@@ -1119,7 +1116,7 @@ Content-Type: application/jose+json
 }
 ~~~~~
 
-When a CA receives a new-registration request containing an
+When a CA receives a new-account request containing an
```

Comment:
I avoid re-wrapping unless a paragraph is very badly out of alignment, because it destroys `git blame`. It also makes diffs harder to read. Any objection?